ginga/qtw/GingaQt.py

```diff
@@ -633,7 +633,11 @@ def new_ws_cb(self, w, rsp, b, names):
     def gui_load_file(self, initialdir=None):
         if self.filesel.exec_():
             fileNames = list(map(str, list(self.filesel.selectedFiles())))
-            self.load_file(fileNames[0])
+            image = self.load_file(fileNames[0])
```

Comment:
@rupak0577, because of the location of the detection code here, it will only open the MultiDim plugin if the file is opened via the File menu.